e2e/utils.go

```diff
@@ -1207,3 +1211,43 @@ func addTopologyDomainsToDSYaml(template, labels string) string {
 	return strings.ReplaceAll(template, "# - \"--domainlabels=failure-domain/region,failure-domain/zone\"",
 		"- \"--domainlabels="+labels+"\"")
 }
+
+// createCustomConfigMap provides multiple clusters information.
+func createCustomConfigMap(pluginPath string, c kubernetes.Interface, clusterID1, clusterID2 string) {
+	path := pluginPath + configMap
+	cm := v1.ConfigMap{}
+	err := unmarshal(path, &cm)
+	Expect(err).Should(BeNil())
+
+	// get mon list
+	mons := getMons(rookNamespace, c)
+	conmap := []util.ClusterInfo{
+		{
+			ClusterID: clusterID1,
+			Monitors:  mons,
+		},
+		{
+			ClusterID: clusterID2,
+			Monitors:  mons,
+		}}
+	conmap[0].CephFS.SubvolumeGroup = "subvolgrp1"
+	conmap[1].CephFS.SubvolumeGroup = "subvolgrp2"
+	data, err := json.Marshal(conmap)
+	Expect(err).Should(BeNil())
+	cm.Data["config.json"] = string(data)
+	cm.Namespace = cephCSINamespace
+	// since a configmap is already created, update the existing configmap
+	_, updateErr := c.CoreV1().ConfigMaps(cephCSINamespace).Update(context.TODO(), &cm, metav1.UpdateOptions{})
+	Expect(updateErr).Should(BeNil())
+}
+
+// validateSubvolumegroup validates whether subvolumegroup is present.
+func validateSubvolumegroup(f *framework.Framework, subvolgrp string) error {
+	cmd := fmt.Sprintf("ceph fs subvolumegroup getpath myfs %s", subvolgrp)
+	_, err := execCommandInToolBoxPod(f, cmd, rookNamespace)
```

Comment:
need to validate the stdout, to make sure the subvolume getpath returns the group path